script.js

```diff
@@ -62,14 +80,24 @@
 // =======================================================================
 
 
+/*
+var x= Math.floor(Math.random() * 10) + 1  
+while(true)
+{
+if(x == y)
```

Comment:
I like that you thought of giving the user a chance to re-try the guessing part here.

It works, although a tip for improving this bit - the variable y is undeclared at line 87 the first time this runs. In stricter languages or other flavours of Javascript, this can throw an error. Next, this loop throws an alert for "Try again" even before the user has the chance to make his first guess.... why?